sim/drivers/infinisim/SpiNorFlash.cpp

```diff
@@ -1,12 +1,12 @@
-#include "drivers/SpiNorFlash.h"
-#include <hal/nrf_gpio.h>
-#include <libraries/log/nrf_log.h>
+#include "SpiNorFlash.h"
```

Comment:
absolue import paths please

```suggestion
#include "drivers/infinisim/SpiNorFlash.h"
```